app/controllers/foodcoop/ordergroups_controller.rb

```diff
@@ -1,6 +1,25 @@
 class Foodcoop::OrdergroupsController < ApplicationController
   def index
-    @ordergroups = Ordergroup.undeleted.order('name')
+    sort = if params["sort"]
```

Comment:
i think this is more or less the same code as in `app/controllers/admin/ordergroups_controller.rb`. What do you think about moving the sorting into the model instead, so code in the controller will reduce do somthing like `@ordergroups = Ordergroup.undeleted.params_sort(params["sort"] || "name")`? (if you have a better name than `params_sort`, just use it). Moving it to the model has the additional advantage that it's much much easier to write tests for it.

Reply:
done.